.drone.star

```diff
@@ -432,6 +432,105 @@ config = {
+			'screenShots': True
+		},
+		'webUI-oc10-integration': {
+			'suites': {
+				'IntegrationApp1': [
+					'webUIAccount',
+					'webUICreateFilesFolders',
+					'webUIDeleteFilesFolders',
+					'webUIFavorites',
+					'webUIFiles',
+					'webUIFilesActionMenu',
+					'webUIFilesCopy',
+					'webUIFilesDetails',
+					'webUIFilesList',
+					'webUIFilesSearch',
+					'webUILogin',
+					'webUIPreview',
+					'webUIPrivateLinks',
+					'webUIRenameFiles',
+					'webUIRenameFolders',
+					'webUIRestrictSharing',
+					'webUISharingAcceptShares',
+					'webUISharingAcceptSharesToRoot',
+					# The following suites may have all scenarios currently skipped.
+					# The suites are listed here so that scenarios will run when
+					# they are enabled.
+					'webUIAdminSettings',
+					'webUIComments',
+					'webUITags',
+					'webUIWebdavLockProtection',
+					'webUIWebdavLocks',
+				],
+				'IntegrationApp2': [
+					'webUIMoveFilesFolders',
+					'webUIResharing1',
+					'webUIResharing2',
+					'webUIResharingToRoot',
+					'webUISharingAutocompletion',
+					'webUISharingFilePermissionMultipleUsers',
+					'webUISharingFilePermissionsGroups',
+					'webUISharingFolderAdvancedPermissionMultipleUsers',
+					'webUISharingFolderAdvancedPermissionsGroups',
+					'webUISharingFolderPermissionMultipleUsers',
+					'webUISharingFolderPermissionsGroups',
+					'webUISharingInternalGroups',
+					'webUISharingInternalGroupsEdgeCases',
+					'webUISharingInternalGroupsSharingIndicator',
+					'webUISharingInternalGroupsToRoot',
+					'webUISharingInternalGroupsToRootEdgeCases',
+					'webUISharingInternalGroupsToRootSharingIndicator',
+					'webUISharingInternalUsers',
+					'webUISharingInternalUsersExpire',
+					'webUISharingInternalUsersCollaborator',
+					'webUISharingInternalUsersShareWithPage',
+					'webUISharingInternalUsersSharingIndicator',
+					'webUISharingInternalUsersToRootCollaborator',
+					'webUISharingInternalUsersToRootPreviews',
+					'webUISharingInternalUsersToRootShareWithPage',
+					'webUISharingInternalUsersToRootSharingIndicator',
+					'webUISharingInternalUsersExpireToRoot',
+					'webUISharingInternalUsersToRoot',
+					'webUISharingInternalUsersToRootBlacklisted',
+					'webUISharingPermissionsUsers',
+					'webUISharingPermissionToRoot',
+					'webUISharingPublicBasic',
+					'webUISharingPublicManagement',
+					'webUISharingPublicDifferentRoles',
+					'webUISharingPublicExpire',
+					'webUITrashbinDelete',
+					'webUITrashbinFilesFolders',
+					'webUITrashbinRestore',
+					'webUIUpload',
+				]
+			},
+			'extraEnvironment': {
+				'WEB_UI_CONFIG': '/srv/config/drone/config-oc10-integration-app-openid.json',
+				'SERVER_HOST': 'http://owncloud/index.php/apps/web/index.html',
+				'EXPECTED_FAILURES_FILE': '/var/www/owncloud/web/tests/acceptance/expected-failures-with-oc10-server-oauth2-login-and-web-integration-app.md'
+			},
+			'filterTags': 'not @skip and not @skipOnOC10 and not @openIdLogin and @smokeTest',
```

Comment:
why `not @openIdLogin ` ? this is using oauth2 or openid?

Reply:
this is using oauth2 for user authentication

Reply:
OK, here using that tag makes sense because we don't want to run the tests that are specific to `openIdLogin`